### PR TITLE
Feature/payment steps select pfs path

### PR DESCRIPTION
**Pull request by nephix**
Adds new payment stepper #402

### CODE REVIEW

**Comment by nephix** [comment]
@kelsos sorry for the large pull request, but most of the changes are just refactoring and tests.

File: raiden-dapp/src/views/TransferSteps.vue [hunk 1]
```diff
@@ -0,0 +1,357 @@
...
+    }
+
+    if (this.step === 3 && this.selectedRoute && this.selectedPfs) {
+      this.transfer();
+    }
+  }
+
+  get token(): Token {
+    const { token: address } = this.$route.params;
+    return this.$store.state.tokens[address] || ({ address } as Token);
```
Comment (kelsos) [inline raiden-dapp/src/views/TransferSteps.vue]: I noticed in my PR that when you have parameters in the URL and this causes an error during loading due to the fact that the default token we use doesn't have any decimals. 

It's not part of this PR but can you try refreshing the payment screen to verify that your don't get any console.errors?

Comment (nephix) [reply raiden-dapp/src/views/TransferSteps.vue]: Thanks 🙏 
Sure will check it out tomorrow and create a follow up ticket if needed
